src/Executables/ExportCoordinates/ExportCoordinates.hpp

```diff
@@ -195,6 +197,50 @@ struct FindGlobalMinimumGridSpacing {
 };
 }  // namespace Actions
 
+namespace OptionTags {
+struct UseDgGrid {
+  using type = bool;
+  static constexpr Options::String help{
+      "Whether to use DG spacing for grid points in an element or FD spacing. "
+      "This will be applied to all elements in the domain. If want to export "
+      "the FD grid, the number of grid points in each element will be 2N-1, "
+      "rather than N."};
+  using group = SpatialDiscretization::OptionTags::SpatialDiscretizationGroup;
+};
+}  // namespace OptionTags
+
+namespace Tags {
+struct UseDgGrid : db::SimpleTag {
+  using type = bool;
```

Comment:
Instead of a bool please use the ActiveGrid enum.  I would find this to be much clearer.

Reply:
I didn't know that existed, thanks for letting me know!